test/functionalTest/cases/1156_mqfilters_and_compounds/mqfilters_and_compounds_existence_subs.test

```diff
@@ -25,7 +25,6 @@ String filters for compound values of attributes: existence in subscriptions
 
 --SHELL-INIT--
 dbInit CB
-#brokerStart CB 0 IPv4 -noCache
```

Comment:
An old leftover... in theory no longer needed as the jenkins test harness pass now run twice (for no-cache an cache cases).

NTC (just informative).